data/systemd/snapd.refresh.service.in

```diff
@@ -7,5 +7,5 @@ Documentation=man:snap(1)
 
 [Service]
 Type=oneshot
-ExecStart=@bindir@/snap refresh
+ExecStart=/bin/sh -c 'if ! @bindir@/snap refresh --list|grep "schedule:.*managed"; then @bindir@/snap refresh; fi'
```

Comment:
that should be --time, not --list, no?

Reply:
Yes and it shows this needs some sort of test :/